### PR TITLE
split toolbar into smaller more manageable components

### PR DESCRIPTION
**Pull request by benouaer**
### Change description ###
splitting the toolbar into smaller more manageable components
**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: projects/media-viewer/src/lib/toolbar/findbar.component.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { Component, Input } from '@angular/core';
+import { ActionEvents, SearchOperation } from '../media-viewer/media-viewer.model';
+
+@Component({
+  selector: 'app-findbar',
+  template: `
```
Comment (linusnorton) [inline projects/media-viewer/src/lib/toolbar/findbar.component.ts]: This seems like a lot for an inline template - how come you didn't use a file?

Comment (benouaer) [reply projects/media-viewer/src/lib/toolbar/findbar.component.ts]: yes, tbh, what I really wanted was to have templates that you can just include. but if it looks untidy, I can create separate html files

Comment (linusnorton) [reply projects/media-viewer/src/lib/toolbar/findbar.component.ts]: Personally I would find it easier to modify the template if it was in it's own file

Comment (benouaer) [reply projects/media-viewer/src/lib/toolbar/findbar.component.ts]: I agree tbh, I'll change it

**Pull request merged**